src/components/message/message.scss

```diff
@@ -0,0 +1,60 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Display message and close button in a centered row.
+ * 2. Push message and close button as far apart as possible.
+ * 3. Condense line height in case it breaks to multiple lines.
+ *
+ * @TODO
+ * 1. Style `:focus` state
+ */
+
+.c-message {
+  align-items: center; /* 1 */
+  background: colors.$gray-light;
+  border-top: 4px solid colors.$gray-dark;
```

Comment:
Can I do something "smarter" than `4px` here? Note to self to look at design token options.

Reply:
Can you explain the design thinking behind the top border? It feels strange to me.

Reply:
Sure. My intent was to add some visual emphasis to the pattern. 

That being said I'm not too attached to it and will be happy to remove them. 

@tylersticka